frontend/pages/AddNewDataset.js

```diff
@@ -17,18 +17,20 @@ export default function AddNewDataset({ prevLoggedIn, prevPublisherId }) {
   };
 
   return (
-    <Grid>
+    <div>
       {JSON.parse(prevPublisherId) <= 0 ? (
         <div>
           <p>No access, please log into a municipality user</p>
         </div>
       ) : (
-        <Grid
-          container
-          spacing={1}
-          direction="column"
-          alignItems="center"
-          style={{ minHeight: '70vh', minWidth: '60vh', marginTop: '5vh' }}
+        <div
+          style={{
+            minHeight: '70vh',
+            margin: '50px 0',
```

Comment:
Using pixels is less responsive, but since this is not a big project, I guess it's fine

Reply:
Less responsive, but more consistent and easier to control, especially with margins and paddings.